src/CommunityToolkit.Maui.Core/Handlers/DrawingView/DrawingViewHandler.android.cs

```diff
@@ -0,0 +1,106 @@
+	/// <param name="view">An instance of <see cref="IDrawingView"/>.</param>
+	public static void MapLines(DrawingViewHandler handler, IDrawingView view)
+	{
+		UpdateLines(view, handler.PlatformView);
+	}
+
+	/// <summary>
+	/// Action that's triggered when the DrawingView <see cref="IDrawingView.ClearOnFinish"/> property changes.
+	/// </summary>
+	/// <param name="handler">An instance of <see cref="DrawingViewHandler"/>.</param>
+	/// <param name="view">An instance of <see cref="IDrawingView"/>.</param>
+	public static void MapClearOnFinish(DrawingViewHandler handler, IDrawingView view)
+	{
+		handler.PlatformView.SetClearOnFinish(view.ClearOnFinish);
+	}
+
+	/// <summary>
+	/// Action that's triggered when the DrawingView <see cref="IDrawingView.LineColor"/> property changes.
+	/// </summary>
+	/// <param name="handler">An instance of <see cref="DrawingViewHandler"/>.</param>
+	/// <param name="view">An instance of <see cref="IDrawingView"/>.</param>
+	public static void MapLineColor(DrawingViewHandler handler, IDrawingView view)
+	{
+		handler.PlatformView.SetLineColor(view.LineColor);
+	}
+
+	/// <summary>
+	/// Action that's triggered when the DrawingView <see cref="IDrawingView.LineWidth"/> property changes.
+	/// </summary>
+	/// <param name="handler">An instance of <see cref="DrawingViewHandler"/>.</param>
+	/// <param name="view">An instance of <see cref="IDrawingView"/>.</param>
+	public static void MapLineWidth(DrawingViewHandler handler, IDrawingView view)
+	{
+		handler.PlatformView.SetLineWidth(view.LineWidth);
+	}
+
+	/// <summary>
+	/// Action that's triggered when the DrawingView <see cref="IDrawingView.MultiLineMode"/> property changes.
+	/// </summary>
+	/// <param name="handler">An instance of <see cref="DrawingViewHandler"/>.</param>
+	/// <param name="view">An instance of <see cref="IDrawingView"/>.</param>
+	public static void MapMultiLineMode(DrawingViewHandler handler, IDrawingView view)
+	{
+		handler.PlatformView.SetMultiLineMode(view.MultiLineMode);
+	}
+	/// <inheritdoc />
+	protected override void ConnectHandler(MauiDrawingView nativeView)
+	{
+		base.ConnectHandler(nativeView);
+		nativeView.Initialize();
+		VirtualView.Lines.CollectionChanged += OnLinesCollectionChanged;
+		PlatformView.DrawingLineCompleted += OnPlatformViewDrawingLineCompleted;
+	}
+
+	/// <inheritdoc />
+	protected override void DisconnectHandler(MauiDrawingView nativeView)
+	{
+		PlatformView.DrawingLineCompleted -= OnPlatformViewDrawingLineCompleted;
+		VirtualView.Lines.CollectionChanged -= OnLinesCollectionChanged;
+		nativeView.CleanUp();
+		base.DisconnectHandler(nativeView);
+	}
+
+	/// <inheritdoc />
+	protected override MauiDrawingView CreatePlatformView() => new(Context);
+
+	void OnPlatformViewDrawingLineCompleted(object? sender, MauiDrawingLineCompletedEventArgs e)
+	{
+		// TODO we need somehow to map from platform view to virtual view (MauiDrawingLine -> ILine)
```

Comment:
I didn't find a way to map from platform view to virtual view. is it possible? as a workaround, we can pass properties instead of interface